test/experimental/test_vectors.py

```diff
@@ -135,3 +140,41 @@ def test_vectors_from_file(self):
         self.assertEqual(vectors_obj['a'], expected_tensorA)
         self.assertEqual(vectors_obj['b'], expected_tensorB)
         self.assertEqual(vectors_obj['not_in_it'], expected_unk_tensor)
+
+    def test_fast_text(self):
+        # copy the asset file into the expected download location
+        asset_name = 'wiki.en.vec'
+        asset_path = get_asset_path(asset_name)
+
+        with tempfile.TemporaryDirectory() as dir_name:
+            data_path = os.path.join(dir_name, asset_name)
+            shutil.copy(asset_path, data_path)
+            vectors_obj = fast_text(root=dir_name)
+
+            # The first 3 entries in each vector.
+            expected_fasttext_simple_en = {
+                'the': [-0.065334, -0.093031, -0.017571],
+                'world': [-0.32423, -0.098845, -0.0073467],
+            }
+
+            for word in expected_fasttext_simple_en.keys():
+                self.assertEqual(vectors_obj[word][:3], expected_fasttext_simple_en[word])
+
+    def test_glo_ve(self):
+        # copy the asset file into the expected download location
+        asset_name = 'glove.840B.300d.zip'
+        asset_path = get_asset_path(asset_name)
```

Comment:
Is this cached?

Reply:
I am storing this in the assets folder I believe